core/src/main/java/com/yahoo/ycsb/generator/SequentialGenerator.java

```diff
@@ -0,0 +1,62 @@
+ * may not use this file except in compliance with the License. You                                                                                                                
+ * may obtain a copy of the License at                                                                                                                                             
+ *                                                                                                                                                                                 
+ * http://www.apache.org/licenses/LICENSE-2.0                                                                                                                                      
+ *                                                                                                                                                                                 
+ * Unless required by applicable law or agreed to in writing, software                                                                                                             
+ * distributed under the License is distributed on an "AS IS" BASIS,                                                                                                               
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or                                                                                                                 
+ * implied. See the License for the specific language governing                                                                                                                    
+ * permissions and limitations under the License. See accompanying                                                                                                                 
+ * LICENSE file.                                                                                                                                                                   
+ */
+
+package com.yahoo.ycsb.generator;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Generates a sequence of integers 0, 1, ...
+ */
+public class SequentialGenerator extends NumberGenerator {
+  final AtomicInteger counter;
+  int _interval,_countstart;
+
+  /**
+   * Create a counter that starts at countstart
+   */
+  public SequentialGenerator(int countstart, int countend) {
+    counter = new AtomicInteger();
+    setLastValue(counter.get());
+    _countstart = countstart;
+    _interval = countend - countstart + 1;
+  }
+  
+  /**
+   * If the generator returns numeric (integer) values, return the next value as an int. Default is to return -1, which
+   * is appropriate for generators that do not return numeric values.
+   */
+  public int nextInt() {
+    int ret = _countstart + counter.getAndIncrement()%_interval;
```

Comment:
For example, you have white space on either side of the '=' here but not on the either side of the '%' operator. This is a 'nit'. Not important. But you asked (smile)